backend/layers/common/entities.py

```diff
@@ -223,6 +223,14 @@ def strip_fields(self):
         if self.consortia is not None:
             self.consortia = [consortium.strip() for consortium in self.consortia]
 
+    def sortConsortia(self):
+        if self.consortia is not None:
+            self.consortia = sorted(self.consortia)
```

Comment:
Nit: you can do `self.consortia.sort()` and will sort in place

Reply:
Updated!